src/frontend/src/optimizer/plan_node/logical_expand.rs

```diff
@@ -147,29 +162,8 @@ impl fmt::Display for LogicalExpand {
 }
 
 impl ColPrunable for LogicalExpand {
-    fn prune_col(&self, required_cols: &[usize]) -> PlanRef {
-        let pos_of_flag = self.input.schema().len();
-        let input_required_cols = required_cols
-            .iter()
-            .copied()
-            .filter(|i| *i != pos_of_flag)
-            .collect_vec();
-        let new_input = self.input.prune_col(&input_required_cols);
-        let input_col_change = ColIndexMapping::with_remaining_columns(
-            &input_required_cols,
-            self.input.schema().len(),
-        );
-        let (new_expand, col_change) = self.rewrite_with_input(new_input, input_col_change);
-
-        let exprs = required_cols
-            .iter()
-            .map(|col| {
-                let mapped_col = col_change.map(*col);
-                let data_type = new_expand.base.schema.fields[mapped_col].data_type();
-                InputRef::new(mapped_col, data_type).into()
-            })
-            .collect_vec();
-        LogicalProject::create(new_expand.into(), exprs)
+    fn prune_col(&self, _required_cols: &[usize]) -> PlanRef {
+        todo!("prune_col of LogicalExpand is not implemented yet.");
```

Comment:
Could this cause crash for some queries?

Reply:
At least not for now.
We may reconsider how to prune_col for expand when group set is added.